Containerfile.redis

```diff
@@ -0,0 +1,33 @@
+# Use Red Hat Universal Base Image (UBI) 9 as the base image
+FROM registry.access.redhat.com/ubi9/ubi-minimal
+
+# Install necessary dependencies (gcc, make, etc.) for building Redis from source
+RUN microdnf install -y gcc make wget tar gzip && \
+    microdnf clean all
+
+# Set the Redis version
+ARG REDIS_VERSION=7.2.4
+
+# Download and extract Redis source code
+RUN mkdir -p /etc/redis && \
+    wget https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
+    tar xzf redis-${REDIS_VERSION}.tar.gz -C /etc/redis --strip-components=1 && \
+    rm redis-${REDIS_VERSION}.tar.gz && \
+    chmod 777 /var/run
+
+# Change working directory to the extracted Redis source code directory
+WORKDIR /etc/redis
+
+# Build Redis from source
+RUN make && \
+    make install
```

Comment:
how long does this take to run?

Reply:
it takes around 1-2 mins.

Reply:
ok, please create a follow up ticket on this, we are definitely going to want something we control more tightly than "Download source from the internet and build it", but this is good enough as a starting point.

Reply:
follow up ticket created: https://issues.redhat.com/browse/OLS-312